daemon/graphdriver/vfs/driver.go

```diff
@@ -70,12 +70,16 @@ func (d *Driver) Cleanup() error {
 
 // CreateReadWrite creates a layer that is writable for use as a container
 // file system.
-func (d *Driver) CreateReadWrite(id, parent, mountLabel string, storageOpt map[string]string) error {
-	return d.Create(id, parent, mountLabel, storageOpt)
+func (d *Driver) CreateReadWrite(id, parent, mountLabel string, storageOpt map[string]string, shared bool) error {
+	return d.Create(id, parent, mountLabel, storageOpt, shared)
 }
 
 // Create prepares the filesystem for the VFS driver and copies the directory for the given id under the parent.
-func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]string) error {
+func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]string, shared bool) error {
+	if shared {
+		return graphdriver.CreateSharedNotSupported("vfs")
```

Comment:
I love this PR and it would be much better if it supports VFS as well. (maybe another PR?)
Then VFS can be useful for production.